uaclient/messages/__init__.py

```diff
@@ -1028,15 +1028,41 @@ class TxtColor:
 
 FIPS_UPDATES_TITLE = "FIPS Updates"
 FIPS_UPDATES_DESCRIPTION = (
-    "NIST-certified core packages with priority security updates"
+    "NIST-certified FIPS crypto packages with priority security updates"
 )
 FIPS_UPDATES_HELP_TEXT = """\
-fips-updates installs fips modules including all security patches for those
-modules that have been provided since their certification date. You can find
-out more at {url}""".format(
+fips-updates installs FIPS 140 crypto packages including all security patches
+for those modules that have been provided since their certification date.
+You can find out more at {url}""".format(
     url=urls.FIPS_HOME_PAGE
 )
 
+FIPS_PREVIEW_TITLE = "FIPS Preview"
+FIPS_PREVIEW_DESCRIPTION = "crypto package under certification with NIST"
```

Comment:
I suggested this in chat as well
```suggestion
FIPS_PREVIEW_DESCRIPTION = "Preview of FIPS crypto packages undergoing certification with NIST"
```